sys/log/full/include/log/log.h

```diff
@@ -531,11 +660,10 @@ int log_read_hdr(struct log *log, const void *dptr, struct log_entry_hdr *hdr);
  * @brief Reads the header length
  *
  * @param hdr Ptr to the header
- * 
+ *
  * @return Length of the header
  */
-uint16_t
-log_hdr_len(const struct log_entry_hdr *hdr);
+uint16_t log_hdr_len(const struct log_entry_hdr *hdr);
```

Comment:
This PR has a lot of unrelated code changes (e.g. code style changes) which should be removed. You can create separate PR for those changes.